### ISSUE TITLE
List of custom LUTs

### ISSUE THREAD

**Issue by devnoname120** [devnoname120/vitabright#37]
> It is designed to be a simple upgrade to the default brightness slider.
> It has five extra bright custom values, including maximum capable brightness of the screen.
> Then six values taken from the default slider, including default brightest, default darkest, and every third > value between. (It divided up perfectly)
> Then a darker custom value.
> Then the even darker value taken from the default inactivity dim.
> And finally three extra dark custom values, including the darkest before the screen loses its contrast.

@buzeak: https://github.com/devnoname120/vitabright/issues/36

> Sharing my vitabright lut file. It has 2 night modes - orange and blue filters on lowest brightness, and 3 modes on highest brightness levels

@evillurker: https://github.com/devnoname120/vitabright/issues/31

> This is a gradual and balance brightness reduction with 16 line of code including two night mode, Flux-like Orange tone, easy on the eyes and Flux-like Ember tone, sleepy mode but can still play

@PeL-H1N1: https://github.com/devnoname120/vitabright/issues/18

> Part of my past gamma table was used in one of the releases, but I have modified it further so night mode (the left extreme of the brightness bar, an orange filter) is more dim, as well as the last steps, plus the right side is brighter than default, so there is more contrast between steps mostly

@afdc98: https://github.com/devnoname120/vitabright/issues/14

> I improved the current "gold standard" LUT and fixed its most glaring issues:
> - Inconsistent brightness variation between levels. They should now be pretty evenly spread, except the lowest 4 vs the rest.
> - Ugly and inconsistent white balance shifts between levels. It should now stay at a neutral gray at every brightness level.
> - Very evident black crush and wrong gamma at the lower 4 levels.

> my LUT was developed on two OLED models (with two VERY different panel behaviours, i may add)

@red9350: https://github.com/devnoname120/vitabright/issues/49

**Comment by red9350** [comment]
I also just published [mine](https://github.com/devnoname120/vitabright/issues/49):

> I improved the current "gold standard" LUT and fixed its most glaring issues:
> 
> Inconsistent brightness variation between levels. They should now be pretty evenly spread, except the lowest 3 vs the rest.
> Ugly and inconsistent white balance shifts between levels. It should now stay at a neutral gray at every brightness level.
> Very evident black crush and wrong gamma at the lower 4 levels.

**Comment by devnoname120** [comment]
@red9350 Thanks a lot! What is your OLED screen model? As far as I remember LUTs are actually model-specific

**Comment by buzeak** [comment]
Yeah, my LUT fIle Is perfect for my screen, they are model-specIfIc lIke devnoname120 says (unfortunately I can't tell you my model rn)

**Comment by devnoname120** [comment]
@buzeak Definitely curious on your feedback if you can test @red9350's modified LUT!
